.github/workflows/build.yml

```
@@ -70,26 +70,8 @@ jobs:
           restore-keys: ${{ runner.os }}-${{ github.job }}-helia-
       - run: sudo apt update
       - run: sudo apt install -y libxkbcommon0 libxdamage1 libgbm1 libpango-1.0-0 libcairo2 # dependencies for playwright
-      - uses: actions/checkout@v4
-        with:
-          repository: ipfs/helia
-          fetch-depth: 0
-          path: interop
-          ref: 'main'
-      - name: Checkout latest tag
-        run: |
-          export TAG="$(git tag | grep "^interop-" | sort --version-sort --reverse | head -n1)"
-          echo "Running tests against: $TAG"
-          git checkout "$TAG"
-        working-directory: interop
-      - run: npm install
-        working-directory: interop
-      - run: npm run build
-        working-directory: interop
-      - run: npm install
-        working-directory: interop/packages/interop
-      - run: npm test
-        working-directory: interop/packages/interop
+      - run: npm i @helia/interop
+      - run: helia-interop
```

Either install globally:

```suggestion
      - run: npm i -g @helia/interop
      - run: helia-interop
```

or install locally and use `npx` to figure out pathing:

```suggestion
      - run: npm i @helia/interop
      - run: npx helia-interop
```

or don't install at all:

```suggestion
      - run: npx --package @helia/interop helia-interop
```

You can test with the RC:

```suggestion
      - run: npm i @helia/interop@next
      - run: npx helia-interop
```